lib/backup_restore/restorer.rb

```diff
@@ -436,6 +436,7 @@ def extract_uploads
         FileUtils.mkdir_p("uploads")
 
         tmp_uploads_path = Dir.glob(File.join(@tmp_directory, "uploads", "*")).first
+        return unless tmp_uploads_path.present?
```

Comment:
When possible, we usually prefer the positive condition instead of the negative counterpart which is not always clear at first sight.

```suggestion
        return if tmp_uploads_path.blank?
```

Reply:
Makes sense. 

https://github.com/discourse/discourse/blob/b39a2c3181024081a5a836d647d40dd1a44fdbf7/lib/backup_restore/restorer.rb#L430

I was following the pattern already used in the block, but happy to change it!